app/pages/sections/catalogue-solutions/select-price/controller.js

```diff
@@ -1,3 +1,21 @@
+import { getData } from 'buying-catalogue-library';
 import { getContext } from './contextCreator';
+import { getEndpoint } from '../../../../endpoints';
+import { logger } from '../../../../logger';
 
-export const getSolutionsPricePageContext = async ({ orderId }) => getContext({ orderId });
+export const getSolutionsPricePageContext = async ({
+  orderId, supplierId, accessToken,
+}) => {
+  const solutionsPricingEndpoint = getEndpoint({ endpointLocator: 'getSolutionsPricing', options: { supplierId } });
```

Comment:
`getSolutionsPricing` should be `getSolutionPricing` (prices for only one solution)

Reply:
also in other places change `solutions` to `solution`